transport.validator.mapcss

```diff
@@ -86,39 +87,70 @@ relation[type=route_master][route_master=~/^(bus|coach|train|subway|monorail|tro
 	set pt_route_master;
 }
 
-relation.pt_route[!public_transport:version]
+way <[role="route"] relation.pt_route,
+way <[role="forward"] relation.pt_route,
+way <[role="backward"] relation.pt_route,
+way <[role="reverse"] relation.pt_route,
+way <[role="forward:stop"] relation.pt_route,
+way <[role="backward:stop"] relation.pt_route
```

Comment:
Good idea :+1: 

But the rules are used both for JOSM and for Osmose and it seems to me that the syntax of parent selector and child selector is not handled by Osmose MapCSS (@frodrigo , can you please confirm this please)?

As a result, the rules you want to add won't work in Osmose, so you can add the `-osmoseItemClassLevel: none;` flag on the instead :crying_cat_face:

Reply:
Will do.